### PR TITLE
support environment specific variables for rules

### PR DESCRIPTION
**Pull request by JetJet13**
Changelog
===
- added functionality for rules to support environment specific variables
- updated README

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth0-deploy",
-  "version": "0.1.16",
+  "version": "0.1.17",
```
Comment (ukayani) [inline package.json]: can we leave this change out? the version is auto bumped by a release command.

Comment (JetJet13) [reply package.json]: yea sure, I wasn't aware.

**Comment by ukayani** [comment]
LGTM. 
However, i'm not really maintaining this anymore. Have you considered the official `auth0-deploy-cli` ? 
I can cut a release anyways, if you wish.

**Comment by JetJet13** [comment]
yea please do. We are still using it here @ L1

**Comment by JetJet13** [comment]
Also, thanks for the quick review. 👍 I did not expect it

**Pull request merged**